rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java

```diff
@@ -71,19 +70,13 @@ public PeersInformation(ChannelManager channelManager,
                 .thenComparing(this::comparePeerTotalDifficulty);
     }
 
-    public void reportEventWithLog(String message, NodeID peerId, InetAddress peerAddress, EventType eventType, Object... arguments) {
-        logger.trace(message, arguments);
-        peerScoringManager.recordEvent(peerId, peerAddress, eventType);
+    public void reportEventToPeerScoring(Peer peer, EventType eventType, String message, Object... arguments) {
+        this.peerScoringManager.recordEvent(peer.getPeerNodeID(), peer.getAddress(), eventType, message, arguments);
     }
 
-    public void reportEvent(NodeID peerId, InetAddress peerAddress, EventType eventType) {
-        peerScoringManager.recordEvent(peerId, peerAddress, eventType);
-    }
-
-    public void reportErrorEvent(NodeID peerId, InetAddress peerAddress, String message, EventType eventType, Object... arguments) {
-        logger.trace(message, arguments);
-        failedPeers.put(peerId, Instant.now());
-        peerScoringManager.recordEvent(peerId, peerAddress, eventType);
+    public void reportSyncingErrorToPeerScoring(Peer peer, EventType eventType, String message, Object... arguments) {
```

Comment:
seems this method does not only reports to PeerScoring, but also populates internal `failedPeers` field. Do you think its name could be something like `processSyncingError()`?

Reply:
yeap, agree

Reply:
Done